Documentation/operations/upgrade.rst

```diff
@@ -406,6 +406,10 @@ Deprecated Metrics
 ~~~~~~~~~~~~~~~~~~
 * ``cilium_node_connectivity_status`` is now deprecated. Please use ``cilium_node_health_connectivity_status`` instead.
 * ``cilium_node_connectivity_latency_seconds`` is now deprecated. Please use ``cilium_node_health_connectivity_latency_seconds`` instead.
+* ``doublewrite_identity_crd_total_count`` is now deprecated. Please use ``doublewrite_identity_crd_total`` instead.
```

Comment:
Maybe should've mentioned, in the past we've had a "Removed Metrics" section when metrics are dropped. In this case the metrics are just renamed, so I would put it in a "Renamed Metrics" or similar.

I don't think its accurate to say they're deprecated.

Reply:
Alright, I moved them to the "Changed Metrics" section above